### DISCUSSION TITLE
Getting error on start of nvim

### DISCUSSION THREAD

**Discussion by azhahes** [LazyVim/LazyVim#1328] [Q&A] [answered]
Getting below error message while starting nvim
   Error  19:19:15 notify.error lazy.nvim Lua module not found for config of nvim. Please use a `config()` function instead
Can someone help me resolve this issue?

**Comment by dpetka2001** [comment]
Did you change some file and this error occurred? I'm not able to reproduce with the steps from the [docs](https://www.lazyvim.org/installation) with a fresh Lazyvim starter template.

**Reply by azhahes** [reply]
yes I was using lazyvim for a while. recently many many change, can't track which change is creating this problem. Is there any way to debug this? or any efficient way to backtrack?

**Reply by dpetka2001** [reply]
Did you make any changes to `~/.local/share/nvim/lazy/`?

**Reply by dpetka2001** [reply]
Can you try deleting the directory `~/.local/share/nvim/lazy/` and restart Neovim?

**Reply by azhahes** [reply]
I found the changes, which causing this issue, i modified the `catpuccin` theme plugin which does take effect so I moved to different theme, but the modification was there it seems. But not sure whats wrong with this settings,
```lua
  {
    "catppuccin/nvim",
    opts = {
      transparent_background = true,
      flavour = "latte", -- latte, frappe, macchiato, mocha
      background = { -- :h background
        light = "latte",
        dark = "mocha",
      },
      dim_inactive = {
        enabled = true, -- dims the background color of inactive window
        shade = "dark",
        percentage = 0.15, -- percentage of the shade to apply to the inactive window
      },
    },
  },
```
Anything wrong in this settings?

**Reply by dpetka2001** [reply]
Are these the whole contents of the file? Do you have a `return` statement in the file?

**Reply by dpetka2001** [reply, marked as answer]
Delete `~/.local/share/nvim/lazy/catppuccin/` directory and restart Neovim. Then create a file `~/.config/nvim/lua/plugins/colorscheme.lua` with the following contents
```lua
return {
  {
    "catppuccin/nvim",
    name = "catppuccin",
    opts = {
      transparent_background = true,
      flavour = "latte", -- latte, frappe, macchiato, mocha
      background = { -- :h background
        light = "latte",
        dark = "mocha",
      },
      dim_inactive = {
        enabled = true, -- dims the background color of inactive window
        shade = "dark",
        percentage = 0.15, -- percentage of the shade to apply to the inactive window
      },
    },
  },
}
```
Does this solve your problem?

**Reply by dpetka2001** [reply]
Also just before the line with `opts` add the following `name = "catppuccin",`.

**Reply by azhahes** [reply]
Thanks, @dpetka2001 that solves the problem
